src/test/ui/const-generics/slice-const-param-mismatch.rs

```diff
@@ -1,5 +1,5 @@
 #![feature(const_generics)]
-//~^ WARN the feature `const_generics` is incomplete
+#![allow(incomplete_features)]
```

Comment:
is there a reason you are not testing `min_const_generics` here?

Reply:
ah I think I was having some trouble checking the types so I omitted it for now, but I think I figured out how to